ci-operator/step-registry/ipi/conf/vsphere/customized-resourcepool/ipi-conf-vsphere-customized-resourcepool-commands.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+# shellcheck source=/dev/null
+declare vsphere_cluster
+source "${SHARED_DIR}/vsphere_context.sh"
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+unset SSL_CERT_FILE
+unset GOVC_TLS_CA_CERTS
+
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/resourcepool.yaml.patch"
+
+CLUSTER_NAME="${NAMESPACE}-${UNIQUE_HASH}"
+rsp_name="${CLUSTER_NAME}-rsp"
+govc pool.create ${vsphere_cluster}/Resources/${rsp_name}
+
+cat > "${PATCH}" << EOF
```

Comment:
Actually, 4.9+ CI jobs are always running with specifying custom resource pool, see [code here](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh#L125).

maybe we can only add post-check here, and if resourcePool is defined in install-config, then check it. In deprovision step, just check that cluster machines are removed from resource pool after cluster is destroyed.

Reply:
Good point, agree.